### ISSUE TITLE
[Blog post] Why Accessibility Should Be Your Next Priority – Insights from WordCamp Europe 2024

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#62]
Submitted blogpost from Wendie Huis in't Veld:

Why Accessibility Should Be Your Next Priority – Insights from WordCamp Europe 2024

[EAA.docx](https://github.com/user-attachments/files/22132764/EAA.docx)

**Comment by rianrietveld** [comment]
props @dolgelukkig
